<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxiesWindow</class>
 <widget class="QWidget" name="ProxiesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proxies</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="pingtestButton">
     <property name="text">
      <string>Ping Test</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="updateButton">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QListView" name="proxiesView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
